plugins/quickinspector/quickoverlay.cpp

```diff
@@ -50,6 +50,50 @@ extern Q_GUI_EXPORT QImage qt_gl_read_framebuffer(const QSize &size,
 QT_END_NAMESPACE
 
 namespace GammaRay {
+
+class QQuickItemPropertyCache {
+public:
+    static const QQuickItemPropertyCache &getPropertyCache(QQuickItem *item)
+    {
+        // className() is generated by moc and static -> we can use the pointer as key
```

Comment:
Not true for dynamic types as we encounter them in QML :) But more importantly, why do we need to look up properties per type at all? Are there really types overriding the properties in QQuickItem, or could we just look up the properties on that once and be done with it?